src/jobflow_remote/remote/host/local.py

```diff
@@ -117,6 +117,28 @@ def remove(self, path: str | Path) -> None:
         os.remove(path)
 
     def rmtree(self, path: str | Path, raise_on_error: bool = False) -> bool:
+        """Recursively delete a directory tree on a host.
+
+        This method must be implemented by subclasses of  `BaseHost`.
```

Comment:
Here its not anymore the baseclass so I guess this paragraph should be removed, maybe the first sentence could be "Recursively delete a directory tree on this local host."